src/NosCore.GameObject/Services/Inventory/InventoryService.cs

```diff
@@ -239,7 +239,7 @@ public IItemInstance MoveInPocket(short sourceSlot, PocketType sourceType, Pocke
             }
 
             var sourceInstance = LoadBySlotAndType<IItemInstance>(sourceSlot, sourceType);
-            if (!(sourceInstance is WearableInstance || sourceInstance is SpecialistInstance))
+            if (!(sourceInstance is WearableInstance || sourceInstance is SpecialistInstance && (targetType != PocketType.Equipment || targetType != PocketType.Specialist)))
```

Comment:
useless check.

Reply:
Nope, specialist cards can be moved from Eq to Specialist card inventory

Reply:
yes but it's handle after line 251 252 here we are just checking only specialist card or wearable are going to the next part

Reply:
NVM those line are good but difficult to read you may need to rework this and delete the line 251 252